python/ql/test/experimental/dataflow/import-star/trois.py

```diff
@@ -0,0 +1 @@
+foo = 5
```

Comment:
Giving the variables the same value does not set us up for testing cross-talk... :-)

Reply:
I mean, the test is looking at the actual flow steps, but sure, I can change this to some other integer.

Reply:
Indeed, but I am hoping that at some point we will also run the code..